app/viewmodels/IndexPageHelper.scala

```diff
@@ -115,6 +115,14 @@ class IndexPageHelper @Inject()(p: views.html.components.p,
     }
   }
 
+  def getContentBasedOnLatePaymentPenaltiesFromModel (etmpData: ETMPPayload)(implicit messages: Messages, user: User[_]): Html= {
+    if (etmpData.latePaymentPenalties.get.isEmpty) {
```

Comment:
To avoid None.get exceptions can we add
```suggestion
    if (etmpData.latePaymentPenalties.getOrElse(List.empty[LatePaymentPenalty]).isEmpty) {
```